.travis.yml

```diff
@@ -10,7 +14,8 @@ before_install:
 - cp target/ci-tooling/maven-settings-for-openfire-plugins.xml $HOME/.m2/settings.xml
 before_script:
 - gem install travis
-script: mvn verify -P ci
+script:
+  - TERM="dumb" ./runIntegrationTests -g
```

Comment:
I like the cleanliness of this solution. Can we do this without moving the actual build (`mvn verify -P ci`) into the runIntegrationTests script?

Reply:
Sure, but I deliberately did put it into the `runIntegrationTests` script, so that one could `git clone openfire && ./openfire/runIntegrationTests` (Which does not work currently because of various reasons). Isn't "mvn verify" a no-op if the tests where already run and the source didn't change (it's the case with `gradle` but I am not sure about `mvn`).

Reply:
I think at best `mvn verify` will only re-run tests. In practice, for reasons I've yet to figure out, `mvn verify` has a tendency to re-compile everything for no apparent reason.

Reply:
`mvn verify` is shorthand for "run all Maven phases up to and including 'verify'." This is the command that does the build (I don't believe we actually have anything bound to the 'verify' phase currently, so we might as well have used `mvn package`). In any case, this command is responsible to do the actual building of the project. Your comment suggests that you think that by that stage, the project is already built - that's not the case.

We could experiment with _duplicating_ the command, which generally should result in no-ops (as you describe) - but as the build process is quite elaborated, some plugins might not play nice here. I'm unsure if it's worth trying.